extensions/amp-a4a/rtc-documentation.md

```diff
@@ -141,6 +141,7 @@ The `errorReportingUrl` property is optional. The only available macros are ERRO
 - Yieldbot
 - Kargo
 - Yieldlab
+- T13
```

Comment:
Nit: I know there are out-of-place vendors here, but could you help putting it in alphabetical order?

Reply:
Gladly, cheers